Back/Reciplastk.Gateway/Controllers/CustomerController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Reciplastk.Gateway.Models;
+using Reciplastk.Gateway.Services;
+
+namespace Reciplastk.Gateway.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class CustomerController : ControllerBase
+    {
+        private readonly CustumerService custumerService;
+
+        public CustomerController(CustumerService custumerService)
+        {
+            this.custumerService = custumerService;
+        }
+
+        [HttpGet("Show all customers")]
```

Comment:
Remember, what you put inside the "" is the Value that goes in the URL. Do not include spaces or strange values. 
```suggestion
        [HttpGet("ShowAll")]
```